frontend/src/App.css

```diff
@@ -87,4 +87,27 @@ h1 {
 
 .answer:hover{
     box-shadow: 0px 0px 5px #C28C19;
+    
+}
+
+.good-answer{
+    text-align: center;
+    background-color:darkgreen;
+    color: white;
+    padding: 1rem;
+    font-family: limelight;
+    border : none;
+    border-radius: 10px;
+    box-shadow: 0px 0px 10px lime;
+}
+
+.wrong-answer{
+    text-align: center;
+    background-color: darkred;
+    color: white;
+    padding: 1rem;
+    font-family: limelight;
+    border : none;
+    border-radius: 10px;
+    box-shadow: 0px 0px 10px red;
 }
```

Comment:
lot of repetition here: add a `.answer` block and put the duplicated code inside it